test/razorIntegrationTests/index.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as jest from 'jest';
+import { Config } from '@jest/types';
+import * as path from 'path';
+import { jestIntegrationTestProjectName } from './jest.config';
+
+async function runIntegrationTests() {
```

Comment:
I would like to re-use the same runner that we use in normal integration tests.  This bit was particularly challenging to get right

It should be possible to export this function from one file and have run just call this.